test/zcl-loader-consecutive.test.js

```diff
@@ -135,7 +135,7 @@ test(
       expect(x.length).toEqual(41)
 
       x = await queryZcl.selectAllStrings(db, jsonPackageId)
-      expect(x.length).toEqual(5)
+      expect(x.length).toEqual(4)
```

Comment:
Add another set of tests which checks all the strings here to improve tests for eg x.includes('char_string'), etc